index.html

```diff
@@ -35,10 +35,215 @@ <h1 class="developers-name">Charlie Giorgania</h1>
+          <div class="works-skills">
+            <ul class="skills-list">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="project-intro">
+          <img src="/resources/images/pinkcomputer.png" alt="Pink Computer" />
+        </div>
+        <div class="works-info">
+          <h3>Project name goes here</h3>
+          <div class="works-skills">
+            <ul class="skills-list">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="about-content">
+            <h2 class="main-topic topic-h2">About</h2>
+            <p class="about-paragraph">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ea eiusmod
+            </p>
+          </div>
+          <div class="get-my-cv">
+            <a
+              class="get-my-cv-btn start-colab colab-butn colab-detail"
+              href="#"
+              >Get my CV
+              <img
+                class="download-cv"
+                src="resources/images/getmycv.png"
+                alt="get my cv image"
+            /></a>
+          </div>
+        </div>
+        <div class="languages-box lang-fw-skills-boxes bottom-boxes">
+          <div class="languages-content">
+            <h3 class="lang-fw-skills">Languages</h3>
+            <ul>
+              <li>Javascript(ES5/ES6)</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+        </div>
+        <div class="frameworks-box lang-fw-skills-boxes bottom-boxes">
+          <div class="languages-content">
+            <h3 class="lang-fw-skills">Languages</h3>
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>CSS</li>
+              <li>Capibara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="skills-box lang-fw-skills-boxes bottom-boxes">
```

Comment:
- [ ] For semantic purposes, please kindly make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`